src/conf.c

```diff
@@ -193,18 +194,19 @@ struct {
         regex_t *cre;
 } directives[] = {
         /* string arguments */
+        STDCONF (basicauthrealm, STR, handle_basicauthrealm),
         STDCONF (logfile, STR, handle_logfile),
         STDCONF (pidfile, STR, handle_pidfile),
         STDCONF (anonymous, STR, handle_anonymous),
         STDCONF (viaproxyname, STR, handle_viaproxyname),
         STDCONF (defaulterrorfile, STR, handle_defaulterrorfile),
         STDCONF (statfile, STR, handle_statfile),
         STDCONF (stathost, STR, handle_stathost),
-        STDCONF (xtinyproxy,  BOOL, handle_xtinyproxy),
```

Comment:
please dont shuffle things around that dont need to be changed.

Reply:
I can put it back to old location... no problem.

But you had seen that before it was this way ?

`        /* string arguments */
        STDCONF (logfile, STR, handle_logfile),
        STDCONF (pidfile, STR, handle_pidfile),
        STDCONF (anonymous, STR, handle_anonymous),
        STDCONF (viaproxyname, STR, handle_viaproxyname),
        STDCONF (defaulterrorfile, STR, handle_defaulterrorfile),
        STDCONF (statfile, STR, handle_statfile),
        STDCONF (stathost, STR, handle_stathost),
        STDCONF (xtinyproxy,  BOOL, handle_xtinyproxy),
        /* boolean arguments */
        STDCONF (syslog, BOOL, handle_syslog),
        STDCONF (bindsame, BOOL, handle_bindsame),
        STDCONF (disableviaheader, BOOL, handle_disableviaheader),`
        
.. so a BOOL definition in the "/* string arguments */" section ?

https://github.com/tinyproxy/tinyproxy/blob/master/src/conf.c#L195

I had tried to understand the used stile how the code inside one file is structured / ordered. But i had not really found there a consequent used logic.

What ordering of adding new variables would you prefer in your code ?
Simply append at the very end of the section where a new definition needs to be done in a fully alphabetical unordered way ?

Reply:
it makes sense to group similar options together, but as i said - don't change what doesn't need to be changed as a drive-by change in an unrelated changeset.

Reply:
I put it back to old location ... even if the line is in the wrong headline section